### DISCUSSION TITLE
Is there a way to edit the header of a ObjectView

### DISCUSSION THREAD

**Discussion by geckigek** [netbox-community/netbox#15454] [Help Wanted!]
Hi there,

does anyone know a way to manipulate the header of a ObjectView? Instead of Video I Path Devicetypes in my plugin I want to write VIP Devcietypes. Or if it is not possible in the ObjectView. Is there an example how to do it in a template?

<img width="945" alt="Screenshot 2024-03-16 190012" src="https://github.com/netbox-community/netbox/assets/86916063/42c0fa8b-e5b5-4a9b-ba94-6728f1953f71">

Best regards

Stefan

**Comment by alehaa** [comment]
If you want to edit just the HTML representation, you can override the [`title` block of the template](https://docs.netbox.dev/en/stable/plugins/development/templates/#standard-blocks). For general changing the display name, change the `__str__()` [standard method](https://docs.netbox.dev/en/stable/plugins/development/models/) of your model.
